README.md

```diff
@@ -223,10 +237,66 @@ scikit-learn = ["sklearn"]
 multiple-modules = ["module1", "module2"]
 ```
 
-Caution when using your mapping is advised: The user-defined mapping takes
-precedence over the other resolvers documented above. For example, if the
-mapping file has some stale/incorrect mapping entries, they will _not_ be
-resolved by the Python environment resolver (which is usually more accurate).
+The provided mapping can be complete or partial. In the case where some of the
+dependencies defined in your project configuration cannot be resolved,
+FawltyDeps will attempt to resolve them using the sequence of resolvers 
+illustrated in the diagram above.
+
+Caution when using your mapping is advised: As illustrated in the diagram, the
+user-defined mapping takes precedence over the other resolvers documented
+above. For example, if the mapping file has some stale/incorrect mapping
+entries, they will _not_ be resolved by the Python environment resolver (which
+is usually more accurate).
+
+Note that in the absence of a custom mapping, FawltyDeps will skip this resolver
+to the next one in the sequence, that is, mapping via local Python environments.
+
+#### Identity mapping
+
+When unable to find an installed package that corresponds to a declared
+dependency either via a user-provided mapping or local Python environments,
+FawltyDeps will fall back to one of two strategies. "Identity mapping", which we
+present in this section is the default fallback strategy. We discuss the other
+strategy in the next subsection.
+
+Identity mapping relies on the simplistic assumption that the dependency provides
+a single import of the same name, i.e. it will expect that when you depend on
+`some_package`, then that should correspond to `import some_package` statements
+in your code.
+
+This assumption is correct for many packages and it allows FawltyDeps to
+produce results (albeit sometimes inaccurate ones) when the current Python
+environment does not contain all of your declared dependencies.
+
+To ensure correctness, however, refer to the next subsection outlining the other
+fallback strategy.
+
+#### Mapping by temporarily installing packages
+Your local Python environements might not always have all your project's
+dependencies installed. Assuming that you don’t want to go through the
+bother of installing packages manually, and you also don't want to rely on
+the inaccurate identity mapping as your fallback strategy, you can use the
+`--install-deps` option, introduced in v0.13.0. This will `pip install`
```

Comment:
This release does not exist yet so maybe I should drop this until it's out.

Reply:
The README should avoid talking about version differences. Since this file is part of the codebase, it should simply reflect the current state (i.e. that `--install-deps` already exists):

```suggestion
`--install-deps` option. This will `pip install`
```